.github/workflows/build_windows.yaml

```diff
@@ -154,8 +156,13 @@ jobs:
               run: |
                   yarn ts-node scripts/generate-builder-config.ts ${{ steps.nightly.outputs.config-args }} ${{ steps.esigner.outputs.config-args }}
                   yarn build --publish never -w --config electron-builder.json ${{ steps.config.outputs.build-args }}
-              env:
-                  SIGNTOOL_PATH: "C:/Program Files (x86)/Windows Kits/10/bin/10.0.22000.0/x86/signtool.exe"
+
+            - name: Check app was signed successfully
+              if: inputs.sign != ''
+              # XXX: UPDATE THIS BEFORE WHEN GOING LIVE
+              continue-on-error: true
+              run: |
+                  . "$env:SIGNTOOL_PATH" verify /pa (get-item ./dist/squirrel-windows*/*.exe)
```

Comment:
To update too

Reply:
We're not going live when this merges, we're still using the SSL.com sandbox environment so this check will fail until we finish procurement and get access to the production environment